modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -915,13 +923,10 @@ private void _addObjectFieldColumn(
 	private void _addOrUpdateObjectFieldSettings(
 			ObjectDefinition objectDefinition, ObjectField newObjectField,
 			ObjectField oldObjectField,
+			ObjectFieldBusinessType objectFieldBusinessType,
```

Comment:
I think we need to sort these parameters. What do you think? Which pattern did you follow regarding sorting?

Reply:
Just done!